ambry-commons/src/main/java/com.github.ambry.commons/BlobId.java

```diff
@@ -492,6 +492,30 @@ public String getUuid() {
     return idBuf.array();
   }
 
+  @Override
+  public byte[] getUuidBytesArray() {
+    ByteBuffer uuidBuf;
+    switch (version) {
+      case BLOB_ID_V1:
+      case BLOB_ID_V2:
+      case BLOB_ID_V3:
+      case BLOB_ID_V4:
+      case BLOB_ID_V5:
+        byte[] uuidBytes = getUuid().getBytes();
+        uuidBuf = ByteBuffer.allocate(UUID_SIZE_FIELD_LENGTH_IN_BYTES + (short) uuidBytes.length);
+        uuidBuf.putInt(uuidBytes.length);
```

Comment:
Is the length prefix needed in this case or could you return `uuidBytes` directly? Since there is only one item (the UUID) in the array.

Reply:
Fair point. Length prefix is removed.